shared/js/background/redirect.es6.js

```diff
@@ -102,6 +102,15 @@ function handleRequest (requestData) {
             if (tracker && socialTracker && trackerutils.shouldBlockSocialNetwork(socialTracker.entity, thisTab.site.url)) {
                 if (!trackerutils.isSameEntity(requestData.url, thisTab.site.url) && // first party
                     !thisTab.site.clickToLoad.includes(socialTracker.entity)) {
+                    // Ensure this website isn't in the excluded domains list.
+                    if (socialTracker.data && socialTracker.data.excludedDomains) {
```

Comment:
Can you outline the difference between the check above `trackerutils.shouldBlockSocialNetwork`?
This uses the cache which allows the user to exempt the load later.

Could you explain why this is needed or if a change to the mentioned method would be better. Perhaps another trackerutils method would clean this up if it is needed?

Reply:
I've looked into this again today and to be honest I can't see why this change is necessary. Like you point out, `trackerutils.shouldBlockSocialNetwork` should take care of this and running that function through a debugger it seems to work OK. Since I wrote this before Christmas I might be forgetting some details, but I'm going to close this for now and will re-open if it turns out there's something I'm forgetting.